bsonproto/objectid.go

```diff
@@ -1,3 +1,33 @@
 package bsonproto
 
-type ObjectID struct{}
+import "fmt"
+
+// ObjectID represents BSON scalar type ObjectID.
+type ObjectID [12]byte
+
+// SizeObjectID is a size of the encoding of [ObjectID] in bytes - 12.
```

Comment:
Also, do we need to document constant value at all? feels like a tautology.

Reply:
I think that all top-level symbols should be documented (or at least all _exported_ top-level symbols). But I'm all for using synonyms and other ways of removing tautology. Any suggestions?

Reply:
I was trying to say that `// SizeObjectID is a size of the encoding of [ObjectID].` is more than enough :) 

Feel free to resolve.

Reply:
Ah, then I did exactly that :)